galleries/examples/statistics/histogram_bihistogram.py

```diff
@@ -0,0 +1,45 @@
+"""
+==========
```

Comment:
Title and section delimiter lines need to be the same length as their text or the docs build system will throw a fit. Lines 2, 4, and 17.